src/store/getters/state-getters.ts

```diff
@@ -158,7 +158,7 @@ export const getCreateRulesStep = (state: StateIF): CreateRulesIF => {
 
 /** Is true when the step is valid. */
 export const isRulesValid = (state: StateIF): boolean => {
-  return getCreateRulesStep(state).valid
+  return getCreateRulesStep(state).validationDetail.valid
```

Comment:
Are you sure that `validationDetail` is not null? Maybe you can use optional chaining.

Reply:
OK, I see you initialize this in State Model. Looks like it's OK.